@navikt/core/react/src/guide-panel/GuidePanel.tsx

```diff
@@ -34,22 +35,32 @@ export interface GuidePanelProps extends HTMLAttributes<HTMLDivElement> {
  * ```
  */
 export const GuidePanel = forwardRef<HTMLDivElement, GuidePanelProps>(
-  ({ children, className, illustration, poster, ...rest }, ref) => (
-    <div
-      {...rest}
-      ref={ref}
-      className={cl("navds-guide-panel", className, {
-        "navds-guide-panel--poster": poster === true,
-        "navds-guide-panel--not-poster": poster === false,
-        "navds-guide-panel--responsive-poster": poster === undefined,
-      })}
```

Comment:
You can't remove the old classes (yet) bc. of the old CSS, and bc. it's breaking. You may update the old CSS to use `data-layout` to make it easier to remove the classes later. Please add it to this issue: https://github.com/navikt/team-aksel/issues/657

Reply:
True... didn't think when I saw the data-props suggestion 😅.

Reply:
I decided to add the new attribute selectors to the old CSS as well, so that we don't have to remember it for later (wasn't too bad).

Reply:
Nice. It's still a breaking change though, since [some are overwriting these classes](https://github.com/search?q=org%3Anavikt+navds-guide-panel--+NOT+repo%3Anavikt%2Faksel&type=code).